enterprise/frontend/src/embedding-sdk/components/public/CreateDashboardModal/CreateDashboardModal.tsx

```diff
@@ -13,7 +13,7 @@ import type { Dashboard } from "metabase-types/api";
 import type { State } from "metabase-types/store";
 
 export interface CreateDashboardModalProps {
-  initialCollectionId: SDKCollectionReference;
+  initialCollectionId?: SDKCollectionReference;
```

Comment:
This type is wrong because we already have the default value
https://github.com/metabase/metabase/blob/fd8a4fe8a2e8f7f0d8e3d4d93872e4bb8224dba0/enterprise/frontend/src/embedding-sdk/components/public/CreateDashboardModal/CreateDashboardModal.tsx#L23